fineract-provider/src/main/java/org/apache/fineract/portfolio/delinquency/service/DelinquencyWritePlatformServiceImpl.java

```diff
@@ -243,36 +277,88 @@ public int compare(DelinquencyRange o1, DelinquencyRange o2) {
     }
 
     private boolean isOverlaped(DelinquencyRange o1, DelinquencyRange o2) {
```

Comment:
MINOR: there is a type in the method name. it should be "isOverlapped"

Reply:
Done!